# <a name="HW10title">Homework 10: Due Wednesday, April 17 at 11:00AM</a>

---

## <a name="HW1Inst">Instructions:</a>
---

1. <font color="tomato">Either click the COPY TO DRIVE button or use the menu File/Save/Save a Copy to save your own version of the notebook in your own folder in Google Drive.</font>

  - If you do not have a Google account, you will need to create a Google account in order to create your own individual copy of this notebook.
  - By default, the notebook will be saved in a folder named **Colab Notebooks** in your drive.
  - Feel free to name and store the notebook wherever you like!
v
2. <font color="mediumseagreen">After you have copied the notebook into your Drive, you can begin editing and saving your work.</font>

**Upload your completed assignment into Canvas as a Google Colab (or Jupyter) Notebook with file extension `.ipynb`.**

Upload your file into the Homework 10 assignment in Canvas before <font color="dodgerblue">**11AM on Wednesday, April 17.**</font>

**You must provide supporting work for your all of your answers.**

  - <font color="dodgerblue">That work will include using R code cells.</font>
    - **Feel free to insert as much R code as you like.**
    - You must interpret or explain how the output of a code cell is being utilized to solve the problem.
    - **Code with incomplete or missing explanations of why the output is useful may not receive full credit.**
  - <font color="dodgerblue">Your work will also involve writing explanations and justifications in text cells.</font>
    - **Be sure to type/write out the steps of your work/explanation so I can see how you arrived at your answer.**
  - <font color="dodgerblue">Your work in some parts may involve doing algebra and/or calculus.</font>
    - **You can type supporting math equations and notation with LaTeX in text cells.**
    - The LaTeX used in the Colab notebooks we cover in class provides great examples.
    - Here's a helpful [Quick Reference Guide for LaTeX](https://colab.research.google.com/drive/1voDDmLfHmRWzWQW5yiHfqYsjGPFqdovP?usp=sharing).
    - Here's a helpful [LaTeX math symbol gallery](https://artofproblemsolving.com/wiki/index.php/LaTeX:Symbols).
    - **Learning LaTeX is super useful and might even be enjoyable!**


## Important Academic Policies:
---

- **Be your own work.** Though you may collaborate with others, everyone is responsible for writing up the work in their own way using their own methods. Plagiarism of any form is not tolerated.
- **Be complete.** You must provide all work and/or explanations needed to find the solution. Answers with insufficient or incomplete supporting work may lose credit.
- **Adhere to the Code of Academic Honesty.**
- **Be clearly written (and legible if written).** Your solution to a problem must be clear, written in complete sentences. You may lose credit for work that is unclear or hard to follow.

## <font color="dodgerblue">Let me know if you need help, and **GOOD LUCK!!!**</font>

----

# <a name="HW10q1">Question 1</a>

---

In the 1970s, research<sup>1</sup> by Alfred Sommer on vitamin A and night blindness suggested that vitamin A also reduced childhood death rates. Children were randomly assigned to two groups, one group took a vitamin A pill every day, and children in the other group were given a **placebo** (a pill that had no vitamin A). Below is a two-way table displaying the results of Sommer's research.


|       | Vitamin A | No Vitamin A | Total  |
|-------|-----------|--------------|--------|
| Died  | 101       | 130          | 231    |
| Lived | 12,890    | 12,079       | 24,969 |
| Total | 12,991    | 12,209       | 25,200 |


<br>

<font size=2>1. [www.laskerfoundation.org/rprimers/sommer/saving1.htm](www.laskerfoundation.org/rprimers/sommer/saving1.htm) and *Introduction to the Practice of Statistics*, 4th ed, by David Moore, George. McCabe</font>


## <a name="HW10q1a">Question 1a</a>

---

Find a 95% confidence interval for the difference in the proportion of children that died between the two groups (received vitamin A and did not receive vitamin A) using the Wald confidence interval for the difference between two proportions.

### Solution to Question 1a

---

<br>

**Complete the R code cell below to compute the cutoffs for a 95% Wald Confidence Interval.**

<br>  


In [ ]:
# Calculate the sample proportion for each separately
treat.p <- 101 / 12991  # proportion in treatment (vitamin A) group that died
control.p <- 130 / 12209  # proportion in control (no vitamin A) group that died

# Approximate the SE using Wald Conf Int
wald.se <- sqrt((treat.p * (1 - treat.p) / 12991) + control.p * (1 - control.p) / 12209)

# Compute the z-alpha-over-2 value for 95% Conf Int
z.star <- qnorm(0.975, 0, 1)

# Compute the Margin of Error
moe <- z.star * wald.se

# Compute upper and lower cutoffs for Conf Int
lower.q1 <- treat.p - control.p - moe
upper.q1 <- treat.p - control.p + moe

# Print answers to the screen
lower.q1
upper.q1

[1] -0.005238789

[1] -0.0005077495

## <a name="HW10q1b">Question 1b</a>

---

Find a 95% confidence interval for the difference in the proportion of children that died between the two groups using the `prop.test` function.

### Solution to Question 1b

---

<br>

**Complete `prop.test()` command in the R code cell below to compute the cutoffs for a 95% Confidence Interval.**

<br>

In [ ]:
prop.test(c(101, 130), c(12991, 12209), conf.level = 0.975)$conf.int

[1] -5.657905e-03 -8.863387e-05
attr(,"conf.level")
[1] 0.975

## <a name="HW10q1c">Question 1c</a>

---

Are your solutions from [Question 1a](#HW10q1a) and [Question 1b](#HW10q1b) equal?

### Solution to Question 1c

---

-   If your answers are equal, explain why.
-   If your answers are not equal, explain why not.

The answers are not equal because they are computing estimates in two different ways; the first way is using a normal distribution while the prop.test is using the exact binomial distribution.

<br>  
<br>  

## <a name="HW10q1d">Question 1d</a>

---

Interpret the meaning of your interval from [Question 1b](#HW10q1b) in practical terms. Can we claim that vitamin A reduces the childhood deaths? Explain how you used your confidence interval to determine your answers.


### Solution to Question 1d

---

There is a 95% chance the interval -0.006 to -0.00009 contains the population parameter. This interval doesn't include, which can allow us to conclude that vitamin A does reduce childhood deaths.

<br>  
<br>  


# <a name="HW10q2">Question 2</a>

---

State whether each set of hypotheses is valid for a statistical test.
If not, explain why not. If so, would you use a one-tail or two-tail test.


a.  $H_0: \mu_1 - \mu_2 \ne 0$ and $H_a: \mu_1 -\mu_2 =0$.

b.  $H_0: \mu_1 = \mu_2$ and $H_a: \mu_1 \ne \mu_2$.

c.  $H_0: \bar{x} = 5$ and $H_a: \bar{x} \ne 5$.

d.  $H_0: p = 0.5$ and $H_a: p < 0.25$.

e.  $H_0: \hat{p} = 0.25$ and $H_a: \hat{p} > 0.25$.

f.  $H_0: \mu = 0.65$ and $H_a: \mu > 0.65$.

g.  $H_0: \mu > 150$ and $H_a: \mu = 150$.

h.  $H_0: p_a - p_b =0$ and $H_a: p_a - p_b < 0$.


## Solution to Question 2

---


Enter your answers below.

<br>  

- Test (a): Ho isn't equality and Ha is equality.


- Test (b): Valid, two tail


- Test (c): Needs to be population.


- Test (d): Valid, one tail.


- Test (e): Needs to be population.


- Test (f): Valid, one tail


- Test (g): Ho and Ha should be flipped.


- Test (h): Valid, just a different form than pa = pb, one tail.






# <a name="HW10q3">Question 3</a>

---

In a hypothesis testing $H_0$: $\mu_1-\mu_2=0$ versus $H_a$: $\mu_1 - \mu_2 \ne 0$, which of the following test statistics would have the smallest $P$-value?

- **Support your answer with one sentence explaining how you determined your answer.**

a.  $\bar{x}_1-\bar{x}_2 = -0.25$

b.  $\bar{x}_1-\bar{x}_2 = -2.5$

c.  $\bar{x}_1-\bar{x}_2 = 1.5$

d.  $\bar{x}_1-\bar{x}_2 = 0.15$

## Solution to Question 3

---

Be sure to support your answer with an explanation.

B would have the smallest p value because it deviates from 0 the most, where we're looking for != 0.


<br>  
<br>  



# <a name="HW10q4">Question 4</a>

---

In a hypothesis testing $H_0$: $p_1-p_2=0$ versus $H_a$: $p_1 - p_2 > 0$, which of the following test statistics is the most statistically significant?

-  **Support your answer with one sentence explaining how you determined your answer.**


a.  $\hat{p}_1-\hat{p}_2 = -0.25$

b.  $\hat{p}_1-\hat{p}_2 = -0.05$

c.  $\hat{p}_1-\hat{p}_2 = 0.005$

d.  $\hat{p}_1-\hat{p}_2 = 0.15$



### Solution to Question 4

---

Be sure to support your answer with an explanation.


D would be the most statistically significant because we're looking for > 0 in our Ha, and d is the most positive.

<br>  
<br>  


# <a name="HW10q5">Question 5</a>

---

A casino wants to see if a six-sided die is biased to land on 1. They perform an experiment in which they roll the die 500 times and count the number of times the result is the die lands on 1.



## <a name="HW10q5a">Question 5a</a>

---

Is this a one or two-sided test? Explain your answer in a complete sentence below.

### Solution to Question 5a

---
This is a one sided test because they want to know if a six sided die is more likely to land on one. This excludes the tail that is being less likely to land on one.


<br>  
<br>  

## <a name="HW10q5b">Question 5b</a>

---

Set up the hypotheses for this (is the die biased to land on 1) test **both in words and in terms the parameter $p$ which denotes the theoretical probability**.

### Solution to Question 5b

---

- $H_0$: p = 1/6 : The die is fair to land on one.


<br>

- $H_a$: p > 1/6 : The die is unfair and more likely to land on one



<br>  
<br>  



## <a name="HW10q5c">Question 5c</a>

---

Out of 500 rolls, they find that 109 land on the number 1. What is the observed test statistic? **Use appropriate notation when expressing your answer.**

### Solution to Question 5c

---

p-hat = 109/500 = 0.218

<br>  
<br>  



## <a name="HW10q5d">Question 5d</a>

---

What is the null distribution for this example? What is the distribution, and what are the corresponding parameter value(s) for the distribution?

### Solution to Question 5d

---

The null distribution for this example is the distribution under the assumption that there is 1/6 chance of rolling a one. This would be a binomial distribution for 500 trials.

<br>  
<br>  



## <a name="HW10q5e">Question 5e</a>

---

Calculate the $p$-value of the observed test statistic using the null distribution you identified in the [Question 5d](#HW10q5d).

### Solution to Question 5e

---

Use the R code cell below to answer the question. No further work or explanation required.


<br>  
<br>  

In [5]:
# use null dist to compute P-value
1 - pbinom(108, 500, 1/6)

[1] 0.001720608

## <a name="HW10q5f">Question 5f</a>

---

Interpret the practical meaning of the $p$-value you found in [Question 5e](#HW10q5e). Write out a complete sentence that explains what the $P$-value is measuring.

### Solution to Question 5f

---

There is a 0.172% chance of rolling a one 109 or more times out of 500, and based on our small p value, it is likely the die is more likely to land on one, and doesn't seem to be a fair die when rolling one.



<br>  
<br>  


# <a name="HW10q6">Question 6</a>

---

Suppose you do an experiment as follows. Five volunteers wish to quit smoking cigarettes. They are randomly split into two groups.

-   Two people are placed into a reward only group.
    -   If after six months they have successfully quit, they receive a $100 reward.
    -   If they are not successful, no penalty.
-   The other three people make a $50 deposit.
    -   If after six months they have successfully quit, they receive a reward of $150.
    -   If they have not quit smoking after six months, they lose their $50 deposit.

The results of the experiment are given below. The names are modified so that <font color="dodgerblue">names that begin with $S$ denote they successfully quit</font>, while those that <font color="tomato">begin with $N$ denote they were not successful quitting</font>.

<br>  

| Group  | Result |
|--------|--------|
| Reward Only | <font color="dodgerblue">Sole</font>, <font color="tomato">Nick</font> |
| Deposit  | <font color="dodgerblue">Serge</font>, <font color="tomato">Nino, Nell</font>  |


<br>  

**Researchers want to test to see if the proportion of people that quit is higher when they are rewarded as opposed to when they are penalized by losing a deposit.** Answer the questions below to complete the steps of a hypothesis test to answer this question.



## <a name="HW10q6a">Question 6a</a>

----

Set up the hypotheses for this test.

### Solution to Question 1a

---

**Be sure to express each hypothesis both in words and using appropriate notation.**

- $H_0$: p_reward = p_deposit : Being rewarded vs penalized doesn't affect the proportion of people who quit smoking.


<br>

- $H_a$: p_reward > p_deposit : Rewarded people are more likely to quit smoking than penalized people.



<br>  
<br>  



## <a name="HW10q6b">Question 6b</a>

---

Compute the observed test statistic.

-   *Hint: Find the difference in the proportion of people in each sample that successfully quit smoking after six months.*

### Solution to Question 6b

---

Either show your calculations in the R code cell below or show your work in how you calculated the observed test statistic in the space below.




<br>  
<br>  



In [7]:
# compute observed test statistic
p_diff <- 1/2 - 1/3
p_diff

[1] 0.1666667

## <a name="HW10q6c">Question 6c</a>

---

How many total permutation resamples are possible?
In other words, out of a total of 5 people, how many ways can we select two people to be in the reward only group?

### Solution to Question 6c

---

Either show yoru work in the space below, or use the R code cell below to answer the question.



<br>  
<br>  



In [8]:
# how many possible premutation resamples
choose(5, 2)

[1] 10

## <a name="HW10q6d">Question 6d</a>

---

Type out all possible permutations resamples and calculate the difference in sample proportions for each permutation resample.


### Solution to Question 6d

---

The first couple of rows in the table below have been completed.
Add additional rows and complete the table.
You should have as many samples in your table as you identified in [Question 6c](#HW10q6c).

<br>  
<br>  

| Sample | Reward Only Group | Deposit Group | Test stat |
|--------|-------------------|---------------|-----------|
| 1      | Sole, Nick   | Serge, Nino, Nell   | $1/6$     |
| 2      | Sole, Serge | Nick, Nino, Nell     | $1$     |
| 3      | Nell, Nick  | Serge , Sole, Nino   | $-2/3$    |
| 4     | Sole, Nell                | Serge, Nick, Nino            | $1/6$        |
| 5     | Sole, Nino                | Serge, Nick, Nell            | $1/6$        |
| 6     | Nell, Nino                | Serge, Nick, Sole            | $-2/3$        |
| 7     | Nell, Serge                | Nick, Sole, Nino            | $1/6$        |
| 8     | Serge, Nino                | Sole, Nick, Nell            | $1/6$        |
| 9     | Serge, Nick                | Sole, Nell, Nino            | $1/6$        |
| 10     | Nick, Nino                | Sole, Serge, Nell            | $-2/3$        |


<br>  
<br>  



## <a name="HW10q6e">Question 6e</a>

---

Based on your previous answers, what is the $p$-value of the observed test statistic?

### Solution to Question 6e

---

Show or explain how you computed the $p$-value.

6 values >= 1/6 out of the 9 resamples plus the initial observation so 7/10 = 0.7.


<br>  
<br>  

